src/__tests__/Hug.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { shallow } from 'enzyme'
+import toJson from 'enzyme-to-json'
+import 'jest-styled-components'
+import { Hug, Text, Card, theme } from '..'
+
+const text = <Text.span>Have you seen this <Text.span bold>card</Text.span> I'm hugging?!</Text.span>
+
+describe('Hug', () => {
+  test('renders with border-radius from theme', () => {
+    const json = renderer.create(<Hug theme={theme} />).toJSON()
+    expect(json).toMatchSnapshot()
+    expect(json).toHaveStyleRule('border-radius', theme.radius)
+  })
+
+  test('renders text, icon, and Child', () => {
+    const json = toJson(shallow(
+      <Hug icon='closeLight' text={text} theme={theme}>
+        <Card p={3} bg='white' color='text'>I'm a card within a hug!</Card>
+      </Hug>
+    ))
+    expect(json).toMatchSnapshot()
+  })
+
+  test('renders text when string is provided', () => {
+    const json = toJson(shallow(
```

Comment:
Why are you using shallow here? Can't the same snapshot testing be done with `renderer.create().toJSON()`

```
const obj = (
 <Hug icon='closeLight' text="I've been known to hug a card from time to time" theme={theme}>
      <Card p={3} bg='white' color='text'>I'm a card within a hug!</Card>
 </Hug>
)
const json = renderer.create(obj).toJSON()
expect(json).toMatchSnapshot()
```

Reply:
I don't think we should use enzyme on design system. Our components should be purely presentational and the least interactive at this layer.

Reply:
Yep, just needed to pass `theme` to `Card` to get this working with renderer.

I think there may be some components in design system that will need to accept `onClick`, in which case we may want to use enzyme to simulate a click, but I'll take it out for now since there's no use case in this PR.